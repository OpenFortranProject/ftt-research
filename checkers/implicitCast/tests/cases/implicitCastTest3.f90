program test

  INTEGER :: x
  REAL    :: y
 
  x = 4 
  y = 42.4 + x
  
  print *, x
  print *, y

end
