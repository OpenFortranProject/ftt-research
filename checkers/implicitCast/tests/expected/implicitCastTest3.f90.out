Warning: environment variable COMPASS_PARAMETERS is not set, trying to use compass_parameters of the current path
 Running Prerequisite SgProject
ImplicitCast: /home/dagit/research/checkers/implicitCast/tests/cases/implicitCastTest3.f90:9.6: Implicit cast in Fortran code found:  expr 'x' from integer_4 to real_4
